applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -149,17 +149,17 @@ return view.extend({
 		s.taboption('advanced', form.Value, 'serial_number', _('Announced serial number'))
 		s.taboption('advanced', form.Value, 'model_number', _('Announced model number'))
 
-		o = s.taboption('advanced', form.Value, 'notify_interval', _('Notify interval'))
+		o = s.taboption('advanced', form.Value, 'notify_interval', _('Notify interval'), _('A 900s interval gives notifies with max-age of 1800s, UDA min.'))
 		o.datatype    = 'uinteger'
-		o.placeholder = 30
+		o.placeholder = 900
 
-		o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Clean rules threshold'))
+		o = s.taboption('advanced', form.Value, 'clean_ruleset_interval', _('Clean interval'), _('Interval for cleaning up unexpired port forwards without traffic'))
```

Comment:
Does the interval itself clean? Write more aggressively e.g.
`Interval which defines how often port-forwards are removed`